from-trello-to-merge.md

```diff
@@ -0,0 +1,37 @@
+# How to Contribute to Code for Chicago Website Project
+Here are the steps to guide you through the process of contributing to the CfC Website. 
+
+1. Find a Trello card in the "To-Do" column you want to take on. Assign your name to it by clicking the pencil icon in the top right of the card when you hover over it. Click "Change Member" and add yourself. 
+
+2. Move the card to the "In Progress" Column. 
+
+3. Make a new branch. ```git checkout -b <branch-name>``` Usuallly the branch name is named after the feature you are adding.
+
+4. Do your coding magic. 
+
+5. When you are ready to commit the changes; in your terminal type
+``` git add -A``` to stage your changes and ```git commit -m "Message on what changes you've made"```to make the commit. 
+If this is the first commit you've made to this branch you will need to ```git push --set-upstream origin <branch-name>```
+Otherwise if you've already set the upstream you can just ```git push```
+
+6. Once you are ready to open a pull request go to https://github.com/Code-For-Chicago/code-for-chicago-jekyll/ and if you are lucky there will be a yellow notice offering you to "Compare & pull request".*
```

Comment:
Can you reword this to say that your latest branch will appear with a "Compare & pull request" button, and if it does not appear then you'll need to select "New pull request" and indicate which branch should be merged into master?

Reply:
Ah, I see. There was an asterisk at the end.
I prefer adding all of the info into one step instead. The asterisk is easy to miss.